config/locales/js-en.yml

```diff
@@ -82,6 +83,7 @@ en:
     label_collapse: "Collapse"
     label_collapsed: "collapsed"
     label_collapse_all: "Collapse all"
+    label_committed_at: "committed revision %{revision}"
```

Comment:
@NobodysNightmare While this change wasn't trivial due to date being a directive itself and thus needing to compile that whole string, I've adapted the suggested nested localization in https://github.com/oliverguenther/openproject/commit/1e11547af365d21a7bc8e824f85bf08e6a366370